fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryLockFactory.java

```diff
@@ -244,6 +253,10 @@ private void fileLockClearFlushAndClose(boolean isRecovery) {
             } finally {
                 closed = flushed; // allow close retry
                 closingContext = null;
+                synchronized (clearingLockNowLock) {
+                    // clearing under lock to avoid spotbugsMain's "Inconsistent synchronization" issue.
+                    clearingLockNow = false;
```

Comment:
Clearing this flag is not necessary, right? IIUC we would not be running this again